src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -47,6 +47,23 @@ CaptureOptionsDialog::CaptureOptionsDialog(QWidget* parent)
   ui_->unwindingMethodComboBox->addItem("DWARF", static_cast<int>(CaptureOptions::kDwarf));
   ui_->unwindingMethodComboBox->addItem("Frame pointers",
                                         static_cast<int>(CaptureOptions::kFramePointers));
+
+  QObject::connect(ui_->unwindingMethodComboBox, &QComboBox::currentTextChanged,
+                   ui_->unwindingMethodWidget, [this](const QString& /*unused*/) {
```

Comment:
Don't use "unused", use the actual name of the parameter.

Reply:
BTW, you don't have to specify the parameter at all. Qt can deal with missing parameters in slots.